api/src/main/java/com/epam/pipeline/dao/pipeline/PipelineDao.java

```diff
@@ -30,6 +30,7 @@
 import com.epam.pipeline.dao.DaoHelper;
 import com.epam.pipeline.entity.pipeline.Folder;
 import com.epam.pipeline.entity.pipeline.Pipeline;
+import com.epam.pipeline.entity.pipeline.PipelineType;
```

Comment:
I think changes are also required in `FolderDao` since it loads Pipelines in listing folder methods